src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -169,7 +169,8 @@ CreateCourseForm.propTypes = {
   isCreating: PropTypes.bool,
 };
 
-export default reduxForm({
+const CreateCourseForm = reduxForm({
```

Comment:
I would do `export default CreateCourseForm` and then do a regular export on the base, that way you only have to update imports on the test file

Reply:
👍 Was following the other examples.  I can adjust this.

Reply:
Yeah, that's how @inventhouse and I did it on the `CreateInstructorForm`, but we should have done export default on the connected one

Reply:
Sorry for the confusion

Reply:
No worries.